### DISCUSSION TITLE
When creating a file select which branch based on filename

### DISCUSSION THREAD

**Discussion by badi95** [trapexit/mergerfs#1324] [Q&A]
I currently have 2 disks that I am using for proxmox backups that I'd like to merge into one disk. I'd like to have the all the backups for a certain day alternate between the disks so that if one disk fails I don't loose all my recent backups or all backups of a specific vm. For example everything from even number days to one disk everything from odd number days to another disk.

**Comment by trapexit** [comment]
There is no way to do that through mergerfs. You'd have to add some high level logic to do that that would be extremely niche. Your best bet is simply do backups to the underlying filesystems directly.

**Comment by badi95** [comment]
Would I be able to have a script that would move the files to their respective disks once they are on either disk in a way that wouldn't mess with mergerfs?

**Reply by trapexit** [reply]
Not sure what you mean. If you want to backup to individual disks... just backup to individual disks. I'm not sure what mergerfs has to do with this.
